Lombiq.OrchardCoreApiClient.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -209,6 +229,60 @@ public static class TestCaseUITestContextExtensions
         context.Configuration.TestOutputHelper.WriteLine("Removing the tenant succeeded.");
     }
 
+    private static async Task<string> TestContentCreateAsync(
+        UITestContext context,
+        ApiClient apiClient,
+        ContentItem contentItem)
+    {
+        var response = await apiClient.OrchardCoreApi.CreateOrUpdateContentItemAsync(contentItem);
+        var contentItemIdFromAPI = JsonConvert.DeserializeObject<ContentItem>(response.Content).ContentItemId;
```

Comment:
Use System.Text.Json instead, since OC 2.0 will use that and we shouldn't add new code for JSON.NET now unless necessary, but this isn't.